### PR TITLE
readme: link to the official gitlab installation docs

### PR DESCRIPTION
**Pull request by sameersbn**
closes #1049

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -68,6 +69,10 @@
 
 Dockerfile to build a [GitLab](https://about.gitlab.com/) container image.
 
+GitLab CE is set up in the Docker image using the [install from source](https://gitlab.com/gitlab-org/gitlab-ce/blob/master/doc/install/installation.md) method as documented in the the official GitLab documentation.
+
+For other methods to install GitLab please refer to the [Official GitLab Installation Guide](https://about.gitlab.com/installation/) which includes a [GitLab image](https://gitlab.com/gitlab-org/gitlab-ce/tree/master/docker).
```
Comment (solidnerd) [inline README.md]: I think it should [Docker GitLab image](https://gitlab.com/gitlab-org/gitlab-ce/tree/master/docker).

**Comment by solidnerd** [comment]
Looks better to me 👍

**Comment by sameersbn** [comment]
Thanks for the review @solidnerd, have made the suggested change.
@adamniedzielski Do let us know your thoughts. Would like this to be resolved ASAP.

**Comment by adamniedzielski** [comment]
@sameersbn Looks great to me. Thank you very much! 💚

**Comment by reynold-lariza** [comment]
much better! :)

**Pull request merged**